test/libyul/yulOptimizerTests/unusedFunctionReturnParameterPruner/all_return_unused.yul

```diff
@@ -0,0 +1,25 @@
+{
+    let a, b, c := f(sload(0))
+    function f(d) -> x, y, z
+    {
+       y := mload(d)
+       z := mload(2)
+       sstore(y, z)
+    }
+}
+// ----
+// step: unusedFunctionReturnParameterPruner
+//
+// {
+//     let a, b, c := f_6(sload(0))
+//     function f(d)
+//     {
+//         let x, y, z
```

Comment:
Maybe it would be better to declare them separately?

Reply:
I can do that, however I think it's better to minimize inserting to the front to the vector `FunctionDefinition.body.statements`. Don't you think a single `VariableDeclaration` would be better from this perspective?

Reply:
I would say it is more important that the optimizer can deal nicely with the variables.